packages/orm/entity/ProvisionableEntity.ts

```diff
@@ -1,14 +1,20 @@
 import { BaseEntity } from "./BaseEntity";
 import { Column } from "typeorm";
-import { ProvisioningStatus } from "../../models/ProvisioningStatus";
+
+export enum ProvisioningStatus {
+  PENDING = "pending",
+  IN_PROGRESS = "in_progress",
+  FAILED = "failed",
+  COMPLETE = "complete",
```

Comment:
The values are uppercase now per spec.   Ordered like spec.
```suggestion
  PENDING = "PENDING",
  IN_PROGRESS = "IN_PROGRESS",
  COMPLETE = "COMPLETE",
  FAILED = "FAILED",
```